unicodetools/src/main/resources/org/unicode/text/UCD/AdditionComparisons/232.txt

```diff
@@ -0,0 +1,16 @@
+# Symbol: UAE DIRHAM SIGN (20C3)
+# https://github.com/unicode-org/utc-release-management/issues/232
+
+# Names always differ.
+# Age always differs since these tests are comparing additions to pre-existing characters.
+Ignoring Name Age:
+
+# Ignore the security and IDNA properties, as these are not yet included for provisionally assigned characters.
+Ignoring Confusable_MA Identifier_Status Identifier_Type Idn_Status Idn_Mapping Idn_2008:
+
+Propertywise [\N{SAUDI RIYAL SIGN-20C1}
+              \N{UAE DIRHAM SIGN-20C3}] AreAlike
```

Comment:
```suggestion
Propertywise [\x{20C1} \N{SAUDI RIYAL SIGN}
              \x{20C3} \N{UAE DIRHAM SIGN}] AreAlike
```
Once that is implemented, this should be `\xN{20C1:SAUDI RIYAL SIGN}` from PD UTS61, but I am looking at implementing the new features from that UTS right now, so for now we live with this approximation…

Reply:
Implemented